OSS13 Server/Sources/World/Objects/Control.cpp

```diff
@@ -11,7 +11,7 @@
 
 Control::Control() : 
 	Component("Control"),
-	ui(std::make_unique<ControlUI>(this))
+	isTileClicked(false), ui(std::make_unique<ControlUI>(this))
```

Comment:
[You can initialize class fields in header file with figure braces](https://arne-mertz.de/2015/08/new-c-features-default-initializers-for-member-variables/):

`bool isTileClicked{};`

It's more convinient then initialization list, if you are using at least C++11. We use C++17 🙂